Platform/ElkhartlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -307,7 +307,7 @@ UpdateFspConfig (
     Fspmcfg->TcssItbtPcie2En            = MemCfgData->TcssItbtPcie2En;
     Fspmcfg->TcssItbtPcie3En            = MemCfgData->TcssItbtPcie3En;
     Fspmcfg->TcssXhciEn                 = MemCfgData->TcssXhciEn;
-    Fspmcfg->TcssXdciEn                 = MemCfgData->TcssXdciEn;
+    Fspmcfg->TcssXdciEn                 = 0;

```

Comment:
I think you should have a S0ix enable config in SBL.  And only disable these when S0ix is enabled.

Reply:
The only config is PlatformNvs->LowPowerS0Idle but its initialize in stage2.
Should we create new config for s0ix?

Reply:
Yes.  Otherwise, XDCI will not work by default since it will be always disabled.

Reply:
Can we add the s0ix in platform feature config data?

Reply:
yes,  it is platform specific.  You can add it for your project.